sunpy/visualization/drawing.py

```diff
@@ -194,6 +194,54 @@ def prime_meridian(axes, *, rsun: u.m = R_sun, resolution=500, **kwargs):
     return visible, hidden
 
 
+@u.quantity_input
+def extent(axes, wcs, *, rsun: u.m = R_sun, **kwargs):
+    """
+    Draws the extent as defined by a given `~astropy.wcs.WCS` instance.
+
+    Parameters
+    ----------
+    axes : `astropy.visualization.wcsaxes.WCSAxes`
+        The axes to plot the prime meridian on, or "None" to use current axes.
+    wcs : `~astropy.wcs.wcsapi.BaseLowLevelWCS`
+        The WCS that defines the extent to be drawn.
+    rsun : `~astropy.units.Quantity`
+        Solar radius (in physical length units) at which to draw the solar
+        prime meridian. Defaults to the standard photospheric radius.
```

Comment:
You didn't edit this after copying it, but see also my comment about removing `rsun` as an argument